release.nix

```diff
@@ -17,16 +17,43 @@ let
     ] ++ lib.optionals (reflex-platform.iosSupport) [
       "ghcIosAarch64"
     ];
-    hsPkgs = lib.genAttrs compilers (ghc: let
-      ghc' = reflex-platform.${ghc}.override {
-        overrides = self: super: import ./. self;
-      };
-    in {
-      inherit (ghc') reflex-dom reflex-dom-core;
+    variations = map (v: "reflex-dom" + v) [
+      ""
```

Comment:
This comes from reflex's `release.nix`. We might use this for various flags someday, so keeping it.